WordPress/WordPressTest/PostCardCellTests.swift

```diff
@@ -1,520 +0,0 @@
-import UIKit
-import XCTest
-import Nimble
-
-@testable import WordPress
-
-private typealias StatusMessages = PostCardStatusViewModel.StatusMessages
-
-class PostCardCellTests: CoreDataTestCase {
-    private var postCell: PostCardCell!
```

Comment:
Is there a plan to re-implement (some of) these tests?

Reply:
Yes! I'll add them in the next PR